config/manager/manager.yaml

```diff
@@ -53,5 +53,18 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        env:
+        - name: GITHUB_ORG
+          valueFrom:
+            configMapKeyRef:
+              name: github-config
+              key: GITHUB_ORG
+              optional: true
+        - name: GITHUB_AUTH_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: has-github-token
```

Comment:
Hope you've already added this secret manually to the staging env?

Reply:
Yup 🙂